gilda/grounder.py

```diff
@@ -53,11 +56,22 @@ class Grounder(object):
         - If :class:`dict`, it is assumed to be a grounding terms dict with
           normalized entity strings as keys and :class:`gilda.term.Term`
           instances as values.
+    order :
```

Comment:
My only comment is, should be make "order" a bit more specifically named? Like "namespace_order" or "namespace_priority"?

Reply:
Yes, I like `namespace_priority` or even better `prefix_priority`

Reply:
The rest of the codebases uses namespace to refer to prefix/namespace so I think we should use that here too.

Reply:
Done in 225bf44